flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/main/java/org/apache/flink/cdc/connectors/paimon/sink/v2/PaimonCommitter.java

```diff
@@ -72,15 +72,16 @@ public void commit(Collection<CommitRequest<MultiTableCommittable>> commitReques
             }
             commitRequests.forEach(CommitRequest::signalAlreadyCommitted);
             LOGGER.info(
-                    String.format(
-                            "Commit succeeded for %s with %s committable",
-                            checkpointId, committables.size()));
+                    "Commit succeeded for {} with {} committable",
```

Comment:
Could we add retry number information in this log to help user to troubleshoot problem?